sjb/config/test_cases/test_branch_origin_web_console.yml

```diff
@@ -8,10 +8,12 @@ extensions:
     - type: "script"
       title: "start openshift server"
       repository: "origin"
+      # installing chrome since FF is not running our e2e tests
```

Comment:
My syntax highlighter was happier with the comment outside the script block, thus I moved it up here.  Not sure if YAML cares that much.

Reply:
We might just not have the comment since ultimately we want FF to work, too. This comment will be misleading when we fix that (hopefully soon).

Reply:
agree, ill remove